webpack.libraries.config.js

```diff
@@ -0,0 +1,126 @@
+const path = require('path');
+const webpack = require('webpack');
+const CleanWebpackPlugin = require('clean-webpack-plugin');
+const BundleAnalyzerPlugin = require('webpack-bundle-analyzer').BundleAnalyzerPlugin;
+
+module.exports = (env, argv) => {
+    const config = {
+        context: process.cwd(),
+        resolve: {
+            extensions: ['.js', '.jsx', '.json', '.less', '.css'],
+            modules: [__dirname, 'node_modules']
+        },
+
+        entry: {
+            library: [
+                '@babel/polyfill',
+                'react',
+                'react-dom',
+                'react-router',
+                'react-router-dom',
+                'graphql-tag',
+                'react-apollo',
+                'react-redux',
+                'redux',
+                'rxjs',
+                'whatwg-fetch',
+
+                // JAHIA PACKAGES
+                '@jahia/ui-extender',
+                '@jahia/moonstone',
+                '@jahia/data-helper',
+
+                // DEPRECATED JAHIA PACKAGES
+                '@jahia/design-system-kit',
+                '@jahia/react-material',
+                '@jahia/icons'
+            ]
+        },
+        output: {
+            filename: 'jahia-commons.dll.js',
+            path: path.resolve(__dirname, 'src/main/resources/javascript/apps/'),
```

Comment:
Can you move from apps to commons folder to avoid mixing appshell and commons, and have a cleanWebpackPlugin in eech ?

Reply:
Nice suggestion :)